gen/gen_mpy.py

```diff
@@ -948,7 +946,7 @@ def register_int_ptr_type(convertor, *types):
         lv_obj->user_data = self;
 
         // Register a "Delete" event callback
-        lv_obj_add_event(lv_obj, mp_lv_delete_cb, LV_EVENT_DELETE, NULL);
+        lv_obj_add_event_cb(lv_obj, mp_lv_delete_cb, LV_EVENT_DELETE, NULL);
```

Comment:
Actual (`master` branch) binding refers to [lvgl @ de97dba](https://github.com/lvgl/lvgl/tree/de97dba59b0ed0ff1213ece872d0397c3bd82880) (v9 dev version), in which `lv_obj_add_event` is the method name: https://github.com/lvgl/lvgl/blob/de97dba59b0ed0ff1213ece872d0397c3bd82880/src/core/lv_obj_event.h#L97

If you want to fix it in LVGL 8.x, then create the PR for [release/v8](https://github.com/lvgl/lv_binding_micropython/tree/release/v8) branch.
Your own branch should be created also from `release/v8` branch, as `master` branch contains LVGL v9 (dev) changes.